<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExperimentalMobilityWidget</class>
 <widget class="QWidget" name="ExperimentalMobilityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="ql_num">
     <property name="text">
      <string>&lt;NUM&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="FloatingValueLineEdit" name="qle_value"/>
   </item>
   <item>
    <widget class="QPushButton" name="qpb_remove">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FloatingValueLineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/ui/elementaries/floatingvaluelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
